tests_integration/__tests__/trim.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+const docPrinter = require("../../src/doc/doc-printer");
+const docBuilders = require("../../src/doc/doc-builders");
+
+const printDocToString = docPrinter.printDocToString;
+const concat = docBuilders.concat;
+const line = docBuilders.line;
+const trim = docBuilders.trim;
+const group = docBuilders.group;
+const indent = docBuilders.indent;
+
+// These tests don't use `runPrettier` because `trim` is not used by any
+// bundled parser (only third-party plugins).
+
+describe("trim", () => {
+  test("trims the current line", () => {

```

Comment:
[`test.each()`](https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout) with `.toEqual()` should be better as the output won't be changed.